core/src/main/java/io/bisq/core/dao/compensation/CompensationAmountException.java

```diff
@@ -0,0 +1,12 @@
+package io.bisq.core.dao.compensation;
+
+import org.bitcoinj.core.Coin;
+
+public class CompensationAmountException extends Throwable {
```

Comment:
I think its better to extend from Exception here.

Reply:
Fixed.